### DISCUSSION TITLE
Full (strict) SSL/TLS activation Dependency

### DISCUSSION THREAD

**Discussion by raminrez** [hiddify/Hiddify-Manager#378] [Ideas ایدهها]
نیاز و پیشنهاد برای امکان وارد کردن مقدار `Cloudflare Global API Key` برای فعال کردن حالت `Full (strict) SSL/TLS`
![Screenshot 2023-03-09 at 12 16 41 PM](https://user-images.githubusercontent.com/7345325/223970046-a1d9addf-f2a0-4172-aef6-b8c664ee837a.png)

توضیحی در این مورد این حالت و چگونگی به دست اوردن Global API Key :
`Full (strict)` کمک برزگی میکنه توی جلوگیری از شناسایی دامین و خوندن پکت ها توست فیلترچی

> فعال کردن حالت Full (strict) یک پیش‌نیاز مهم داره که درصورتی که رعایت نشده باشه با حالت Full هیچ فرقی نداره
> پیشنیاز اینکار داشتن Global API Key و استفاده ازش داخل AcmeScript هست٬ موقع صدور گواهی‌نامه های SSL

پیدا کردنش هم خیلی راحته٬ که هر کاربر نیازه خودش حتما از داخل پنل کلادفلر API Key رو برداره
1- صفحه اول دشبورد
![Screenshot 2023-03-09 at 12 18 17 PM](https://user-images.githubusercontent.com/7345325/223972959-87546238-9055-4d5f-9857-a30a9dc57510.png)

2- زدن رو دکمه View از قسمت Global API Key
![Screenshot 2023-03-09 at 12 19 26 PM](https://user-images.githubusercontent.com/7345325/223973044-e6a677c5-4976-49b5-bc29-c94c1ec52def.png)

**Comment by hiddify-com** [comment]
عزیز این واقعا هیچ تاثیری ندااره
این در مورد اتصال خود کلودفلر به سرور هست و اصن توسط فیلترچی دیده نمیشه
نگران نباشید

**Comment by zZedix** [comment]
من دامنه های cdn همین الان همشون روی full strict هستن ! مشکلی هم ندارم
